pkgs/games/shipwright/default.nix

```diff
@@ -78,6 +79,29 @@ stdenv.mkDerivation rec {
     StormLib
   ];
 
+  patches = [
```

Comment:
Could you add a comment explaining these patches are to avoid writing to the nix store / the users current `PWD`? I know the individual patch titles already explain that, but it'd be nice just for quickly glancing over this code in the future.

Reply:
Have these patches been proposed upstream?

Reply:
Doesn't look like it. They seem to be confined to [PaulGrandperrin](https://github.com/PaulGrandperrin)'s personal repository with no corresponding upstream pull requests.